docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless Preview 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# FeatureBase Serverless
```

Comment:
Needs a clearer heading to introduce what will be unknown functionality. For example:

```
# How do I scale compute power and storage for my FeatureBase Database?
```